rollup-unoptimized.config.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-env node */
+import graphqlCompiler from 'rollup-plugin-graphql-js-client-compiler';
+import baseConfig from './rollup-common.config';
+
+baseConfig.plugins.unshift(
+  graphqlCompiler({
+    schema: './schema.json',
+    optimize: false,
+    profileDocuments: ['src/graphql/**/*.graphql']
+  })
+);
+
+baseConfig.targets = [
```

Comment:
since this is just reassigned, `targets` should probably not be part of base and should be promoted to the consumed config.